### ISSUE TITLE
Easy Coding HOWTO

### ISSUE THREAD

**Issue by Popolechien** [openzim/zim-requests#559]
- Website URL: https://www.easycoding.org/category/howto
- License: **Creative Commons Attribution Share Alike **
- Desired ZIM Title: **EasyCoding.org**
- Desired ZIM Description: **Short description**
- Desired ZIM Icon –png (URL or attach one): n/a
- Language (ISO 639-3): **rus**
- Is this a MediaWiki?: **no**

Notes: 
1. Note sure if we can only capture the `howto` category
2. Website footer indicates (c) EasyCoding but the terms & conditions page clarifies `Все размещённые тексты, документы и изображения лицензируются по условиям [Creative Commons Attribution Share Alike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/)`

**Comment by Popolechien** [comment]
@xvitaly any chance you could correct the discrepancy highlighted above re: licensing on your website?

**Comment by xvitaly** [comment]
> any chance you could correct the discrepancy highlighted above re: licensing on your website?

All tests, images, etc. -  `CC-BY-4.0`.
Code fragments - ` GPL-3.0-or-later`.

**Comment by Popolechien** [comment]
Yes I got this bit, I was more referring to the © 2005 - 2022 Официальный сайт EasyCoding Team in the footer
